docs/development/CI/infrastructure.md

```diff
@@ -32,6 +123,7 @@ The testing workflows already use a matrix strategy to run the automated tests o
 
 ## Future improvements
 
+- Reenable code coverage workflows (potentially on CSCS-CI).
 - Split code quality: it might be better to run code quality tools separate for each project in the monorepo.
 - Split documentation: once there is proper HTML documentation generated for the projects, it might make sense to have that run as one job per project.
-- Template for tests: although there is a reusable workflow for the code coverage uploading, it probably make sense to reuse some of the workflow description for the tests as well.
\ No newline at end of file
+- Template for tests: although there is a reusable workflow for the code coverage uploading, it probably make sense to reuse some of the workflow description for the tests as well.
```

Comment:
Is this still up to date? I don't think the code coverage template is still valid or useful. I do agree that we could provide a general tempalte for tests.